### ISSUE TITLE
Timezone 2021a new release: please update moment-timezone

### ISSUE THREAD

**Issue by casper5822** [moment/moment-timezone#919]
The new iana 2020e tzdata is here.
Please update the lib

**Comment by devdemi** [comment]
Are there changes for Europe/Volgograd in the new iana 2020e tzdata?
Europe/Volgograd was +3 UTC became +4 UTC

**Comment by slaeme** [comment]
> Are there changes for Europe/Volgograd in the new iana 2020e tzdata?
> Europe/Volgograd was +4 UTC became +3 UTC

Yes.

You can subscribe to [mailing announcements](https://www.iana.org/time-zones). 
Or you can watch [mailing archive](http://mm.icann.org/pipermail/tz-announce/).

Where the last text contains:

> Briefly: Volgograd switches to Moscow time on 2020-12-27 at 02:00.

**Comment by casper5822** [comment]
The iana 2020f was released but it contains only a bug fix so the db version is equal to the 2020e.

**Comment by khmyznikov** [comment]
Please update to 2020f

**Comment by casper5822** [comment]
It will be cool if moment-timezone will have an indipendent way to update tzdatabase. Something like java tzupdater. That way everyone can install the right version without asking for a new release everytime to the maintainers.

**Comment by casper5822** [comment]
The 2021a release of the tz code and data is available. Any update on timezone ?

**Comment by robin-karlsson** [comment]
I'd love to do a pull request for this. Last time it was rejected without any comment though, does anyone handle merging? If so, I can make a PR for this.

The alternative is to fork and create a separate npm package for quicker updates from tzdb, I guess?

**Comment by casper5822** [comment]
@robin-karlsson yes, i think it could be like a function that get an npm externam tzdb. I know there is already a github project for iana tzdb that you can install with npm:
https://github.com/vvo/tzdb. 
So  something like this.

**Comment by gilmoreorless** [comment]
@casper5822 A feature like that might be less work for the maintainers _once it's available_, but it would definitely be much more work to actually implement it in the first place. Moment Timezone is consumed in many different ways, not just npm, so any separation of data updates would need to account for that.

Like you, I'd love to see something like Java's tzupdater for this library, but given the project's current status I think it's unlikely to happen. Especially since the current primary maintainer seems to be appearing less and less these days.

**Comment by robin-karlsson** [comment]
@gilmoreorless Yes, my basic idea was to maybe let an agent make automatic pull request when new tzdb information appears on a forked version and automatically push to npm + create release.

Seems like we're more people that have some migration work to do before being able to leave moment js fully. And such a solution would buy some time.

**Comment by casper5822** [comment]
@robin-karlsson, in our case migration is something to be aware because we have a marketplace where customers can book events (so timezone is very important).

We read the tips of momentjs teams to use luxon or other libs. We made the migration to luxon but: 
Luxon uses the (limited) Intl API, so it means: no timezone files (cool), but,  because it uses the browser timezone update systems.

This is a good thing but not so good, because you lose the control on timezone updates and it is up to people devices (people must install the OS and browser updates) to get the last version of timezone.
In our case, that means that the control of a critical part moves to client side.

So migration it's a good thing but you have to consider the Intl basic usage. Moment -> Luxon o Moment -> Day.js means this.

In short, I know that this feature can be heavy, so we can only wait for tzdb moment updates, because we can't lose the control of timezones. We will continue to search for a valid alternative lib but it's not easy.

**Comment by gilmoreorless** [comment]
I see the 2021a data was [released in version 0.5.33](https://github.com/moment/moment-timezone/releases/tag/0.5.33), but without any update on this issue. So I guess it can be closed now.
